sdk/python/tests/unit/infra/online_store/test_dynamodb_online_store.py

```diff
@@ -171,3 +173,24 @@ def test_online_read(repo_config, n_samples):
     )
     assert len(returned_items) == len(data)
     assert [item[1] for item in returned_items] == list(features)
+
+
+@mock_dynamodb2
```

Comment:
We're planning on replacing this with the dynamo testcontainer, is that right?

Reply:
I was thinking on using test-containers for integration tests and moto for unit tests. Considering not to increase ci testing time, moto is faster than spinning up a docker container. 

What are your thoughts?

Reply:
That works for me!